static/js/payment.js

```diff
@@ -1,18 +1,14 @@
 (function() {
 
-function plan_cost() {
-  var plan = document.querySelector('input[name=plan]:checked'),
+function price_cost() {
+  var price = document.querySelector('input[name=price]:checked'),
+    pricing = document.getElementById('js-price-information'),
     ctick = document.getElementById('id_charitable_tick'),
     c = document.querySelector('input[name=charitable]:checked');
-  plan = plan ? plan.value : '';
+  price = price ? price.value : '';
   ctick = ctick ? ctick.checked : '';
   c = c ? c.value : '';
-  var num = 20;
-  if (plan === 'mapit-100k-v') {
-    num = 100;
-  } else if (plan === 'mapit-0k-v') {
-    num = 300;
-  }
+  var num = pricing.dataset[price] || 20;
```

Comment:
This was still hard-coded - had to add a minimum price data field, and data lookups are lowercase.